frontend/public/components/build.tsx

```diff
@@ -127,40 +128,52 @@ export const BuildNumberLink = ({ build }) => {
 };
 
 const BuildGraphs = requirePrometheus(({ build }) => {
-  const podName = _.get(build, ['metadata', 'annotations', 'openshift.io/build.pod-name']);
+  const podName = build.metadata.annotations?.['openshift.io/build.pod-name'];
   if (!podName) {
     return null;
   }
-
+  const endTime = build?.status?.completionTimestamp
```

Comment:
We're assuming that `build` is set elsewhere in this function, so I guess we can use `build.` instead of `build.?` everywhere.